datajunction-server/datajunction_server/api/djql.py

```diff
@@ -0,0 +1,91 @@
+"""
+Data related APIs.
+"""
+
+from typing import Optional
+
+from fastapi import APIRouter, Depends, Request
+from sqlmodel import Session
+from sse_starlette.sse import EventSourceResponse
+
+from datajunction_server.api.helpers import build_sql_for_dj_query, query_event_stream
+from datajunction_server.models.query import QueryCreate, QueryWithResults
+from datajunction_server.service_clients import QueryServiceClient
+from datajunction_server.utils import get_query_service_client, get_session
+
+router = APIRouter()
```

Comment:
We're starting to use tags to organize things nicely in the swagger UI so it'd be good to add a tag here. Something like: `router = APIRouter(tags=["djql"])`